### PR TITLE
Update events docs

### PR DESCRIPTION
**Pull request by akkomar**

### CODE REVIEW

File: docs/user/reference/metrics/event.md [hunk 1]
```diff
@@ -382,9 +382,16 @@ A maximum of 50 extra keys is allowed.
...
-* `type`: The type of value this extra key can hold. One of `string`, `boolean`, `quantity`. Defaults to `string`.
+* `type`: The type of value this extra key can hold. One of `string`, `boolean`, `quantity`. Defaults to `string`. Recorded value is converted to string for transmission.
   **Note**: If not specified only the legacy API on `record` is available.
 
+{{#include ../../../shared/blockquote-info.html}}
+
+#### Extras or string metrics?
+
+> When designing your metrics, define properties that are slow-changing and common across all events in a given ping as string metrics.
+> Properties specific to a single event or a subset of events should be defined as event extras.
```
Comment (akkomar) [inline docs/user/reference/metrics/event.md:393]: This comes up often in discussions with users.

File: docs/user/reference/metrics/event.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Each event contains the following data:
 - A timestamp, in milliseconds. The first event in any ping always has a value of `0`, and subsequent event timestamps are relative to it.
   - If sending events in custom pings, see [note](../../user/pings/custom.md#the-gleanrestarted-event) on event timestamp calculation throughout restarts.
 - The name of the event.
-- A set of key-value pairs, where the keys are predefined in the `extra_keys` metric parameter, and the values are strings.
+- A set of key-value pairs, where the keys are predefined in the `extra_keys` metric parameter. Values are one of `string`, `boolean`, `quantity`, and are converted to `string` for transmission.
```
Comment (akkomar) [inline docs/user/reference/metrics/event.md:11]: This was inconsistent with `extras` section below.

File: docs/user/user/metrics/adding-new-metrics.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ For all of the metric types in this section that measure single values, it is es
 
 For tracking user behavior, it is usually meaningful to know the over of events that lead to the use of a feature.  Therefore, for user behavior, an [event metric](../../reference/metrics/event.html) is usually the best choice.
 
-Be aware, however, that events can be particularly expensive to transmit, store and analyze, so should not be used for higher-frequency measurements.
+Be aware, however, that events can be particularly expensive to transmit, store and analyze, so should not be used for higher-frequency measurements - though this is less of a concern in [server environments](../adding-glean-to-your-project/server.md).
```
Comment (travis79) [inline docs/user/user/metrics/adding-new-metrics.md:42]: Hmm, seems like I remember our position on events changing. I wonder if I should file a bug to discuss removing this line from the docs 🤔

**Review by travis79 — APPROVED** [review]
Thank you for clarifying the docs!

File: docs/user/user/metrics/adding-new-metrics.md (hunk 3 above)
Comment (akkomar) [reply docs/user/user/metrics/adding-new-metrics.md:42]: Yeah, maybe?
Some thoughts about areas mentioned here:
* storage - we monitor it, shouldn't be of concern to users, worth checking platform monitoring dashboard where we stand
* analysis - is improved with events_stream
* transmission - I guess depends on batching/not batching

**Pull request merged**